components/definition/definition.hbs

```diff
@@ -0,0 +1,16 @@
+<div class="definition js-details"
+  data-hide-label="{{definitionText}}"
+  data-show-label="{{definitionText}}">
+
+  <a href="#definition"
```

Comment:
With javascript off the link is hidden but there should be a heading displayed in its place.
The link copy can be used for the heading copy. The heading should also be of exactly the same weight and size as the link (using existing heading size classes).

Reply:
This has been added with a new utility class: u-no-js-show

Reply:
I think that it would be more appropriate as:
`<h3 class="venus u-no-js-show u-mb-no">{{definitionText}}</h3>`
so that it is more semantically structured.

Reply:
Should we put a heading tag here if we lack the context of the page in which the component will live in an application? If yes, should we change the link text to be a heading?

Reply:
Whoever implements the component should have a suitable level of knowledge to amend a heading level to match the semantic structure of the page. I think that a heading would be the most appropriate element here as it would also provide document structure that A11y browsers would interpret.

Whether the link should be a heading as well is something I'd take advice on.

@will-madebymoon @hamishtaplin @bennpearson